db/schema.rb

```diff
@@ -81,6 +92,7 @@
   add_foreign_key "bookings", "users"
   add_foreign_key "favourites", "offers"
   add_foreign_key "favourites", "users"
+  add_foreign_key "offers", "tags"
   add_foreign_key "offers", "users"
```

Comment:
This also would need to be updated. Not sure what the best approach is (migration?)